tests/compute/struct-default-init.slang

```diff
@@ -3,9 +3,9 @@
 
 struct Test
 {
-	int a;
```

Comment:
It currently does indeed issue a warning here. The reason is that a default constructor is still created, and within that default constructor the field `a` is not initialized. I think its worth keeping the warning; even if it is not a problem in the current file, it could be a problem when using with modules.

Reply:
I am not sure if we want to print a warning about this case,
```
struct Test
{
    int a;
};
```
C/C++ doesn't do that....

Reply:
That is true. I will see if I can make an exception for this.